### PR TITLE
Buffs SM Events

### PR DESCRIPTION
**Pull request by silverplatedelta**
## What Does This PR Do

Makes previously less impactful SM events more impactful
C/D class events now add MUCH more gas to the engine, heat penalty threshold decease doubled for C3.
B Class events now cause double their original effect
S-Laminate now causes 25x heat output, 25x waste output (not as much as you think) and causes a fairly large EMP pulse upon starting. 

## Why It's Good For The Game

Most of our events you can completely ignore, this makes them slightly more impactful.

## Testing

manually test each modified SM event by commenting out all the others. Cry. regret datumizing them, repeat.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
tweak: Makes some SM events more dangerous
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
Please not the EMP god please no

**Comment by BakKeener** [comment]
How many S-Class Events is there? I think the only one I've seen was the lightning arc one that destroyed the entire area before exploding in the end. The new S-Class one at least make you get N2 Tanks loaded with very cold gas to deal with the insane heat that shows up without having to just pre build grounding rods just so the engine doesn't explode.

**Comment by 1080pCat** [comment]
> ### Declaration
> 
>     * [ ]  I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

i hope you do have per-aproval :^)

**Comment by Fordoxia** [comment]
> How many S-Class Events is there? I think the only one I've seen was the lightning arc one that destroyed the entire area before exploding in the end. The new S-Class one at least make you get N2 Tanks loaded with very cold gas to deal with the insane heat that shows up without having to just pre build grounding rods just so the engine doesn't explode.

There are two S class events. Arc forces EER to go over 8000, causing it to begin delaminating and shooting arcs. Without grounding rods the arcs will cast explosions on anything they hit.

Laminate rejection increases heat and waste gas production.

File: code/modules/power/engines/supermatter/supermatter_event.dm [hunk 1]
```diff
@@ -121,7 +121,7 @@
 	duration = 5 MINUTES
 
 /datum/supermatter_event/charlie_tier/heat_penalty_threshold/on_start()
-	supermatter.heat_penalty_threshold -= -73
+	supermatter.heat_penalty_threshold -= -173
```
Comment (Henri215) [inline code/modules/power/engines/supermatter/supermatter_event.dm:124]: you mentioned this one on the pr description?

File: code/modules/power/engines/supermatter/supermatter_event.dm [hunk 2]
```diff
@@ -137,14 +137,14 @@
...
-	supermatter.gas_multiplier = 1.5
+	supermatter.gas_multiplier = 3
 
 
 /datum/supermatter_event/bravo_tier/heat_multiplier
 	name = "B-2"
 
 /datum/supermatter_event/bravo_tier/heat_multiplier/on_start()
-	supermatter.heat_multiplier = 1.25
+	supermatter.heat_multiplier = 3
```
Comment (Henri215) [inline code/modules/power/engines/supermatter/supermatter_event.dm:147]: This one is not double 😃 

image

Comment (silverplatedelta) [reply code/modules/power/engines/supermatter/supermatter_event.dm:147]: good nuff for government work

**Review by DGamerL — APPROVED** [review]

**Pull request merged**